addons/stock/models/stock_picking.py

```diff
@@ -387,6 +387,12 @@ class Picking(models.Model):
         ('available', 'Available'),
         ('expected', 'Expected'),
         ('late', 'Late')], compute='_compute_products_availability')
+    packaging_reserve_method = fields.Selection([
```

Comment:
@thomasdeleval Is it not better to have this setting of picking type instead of picking itself ? If you want to work with a full reservation by packaging, you have to tick the good option every time.

Reply:
After talk about that with Thomas, he confirms the setting should be on the picking type.